### ISSUE TITLE
Website Refresh Post-Mortem

### ISSUE THREAD

**Issue by llkats** [nodeschool/sanfrancisco#209]
# NodeSchool SF Website Post-Mortem

Following up from https://github.com/nodeschool/sanfrancisco/issues/172

tl;dr: The refactor effort stalled out. I forked NodeSchool Oakland's site as the direction to move forwards. Please help out with polishing that branch so we can get the site updated ASAP. I'm sorry I did not succeed with the original project plan.

## What We Did and What Got Done
We had some initial energy around planning and wireframing, which produced a great wireframe and initial outlines of content. After that, we were able to set up the beginnings of infrastructure and a few initial pieces of markup for site content.
## What Stalled Out
The site did not reach completion of even just basic content layout.
## Why?
- We had a lot of objectives for the site and ended up outlining a complex site without focusing the ideas down into a minimum viable product. People got busy with understandable life stuff, and the project was not organized in a way that could sustain effort for the time that people could donate to it. In short, we bit off more than we could chew.
- I had a goal of breaking up tasks such that anyone could approach them, but  I learned that if you don't go out to find people to make those contributions, then they won't get done. Especially if the tasks are a little more on the tedious side, like writing markup. It was a well-intentioned but ultimately misguided idea.
- I think that without responsive turnaround times on pull requests, the effort quickly lost steam and became stagnant. This failure falls squarely on me. I did not do a good job of empowering other contributors with the ability to merge PRs. I think that if I'd done a better job at making sure other people felt like they could make decisions, then we would not have lost so much momentum.
- Ultimately, we had a project manager who'd never managed a project before (hi, that was me). I learned a lot about delegation, time management, and communication, but this learning occured at the expense of the project.
## What I'd Try Next Time
- Defining the tools/library/setup well ahead of time to avoid getting into the weeds over tool selection.
- Really being honest with myself about the time I could actually donate to an effort like this. At the time, I was ending at a job and starting a new one, and didn't actually have the capacity to be leading a project.
## What Needs Help Moving Forwards
- Tim Kempf of NodeSchool Oakland has worked on automating setup for his events, including scripts to create the necessary GitHub issues and Tito signup pages, as well as dropping links to all of them into Oakland's website. I think that [his site has a simple, information-dense design](https://nodeschool.io/oakland/), and can easily be extended with extra information and features. 
- I forked Tim's design for an initial pass at refactoring NodeSchool SF's site. Here is a screenshot of its progress so far:
![nodeschool_sf](https://user-images.githubusercontent.com/578156/27018621-bf9aa2f8-4ee6-11e7-9bf6-005ffbbaf12c.png)
    - yes, I know it says July 8thth
- [I created a new issue](https://github.com/nodeschool/sanfrancisco/issues/210) for this new direction, and closed all of the old issues.
## Thoughts?
- Please let me know if y'all have any thoughts or questions. I know that this is taking a left turn from something that was communal, but the ultimate goal is to get the site updated, and this is the simplest way I can think of to move things forward quickly. I'm sorry to let everyone down with the initial plan, and hope to do better at organizing similar efforts in the future.
## Thanks
Special thanks to @samarthgulati, @tlhunter, @skipzero, @teone, @paulgrock, @benhalverson, and others for their work and effort on the project. I hope y'all will continue to contribute to NodeSchool SF despite this setback, because we truly couldn't do any of this without you.

**Comment by jedireza** [comment]
You nailed it. I support everything you said and thank you. I don't think you let anyone down. Quite the contrary, NodeSchoolSF and the website are only better off because of the effort you and everyone else put forward. 🙏 

Re: The forked site: Hell yeah! I say let's ship it and open issues and PRs to incrementally change/fix/improve things. 🎉

**Comment by benhalverson** [comment]
I'll continue to help with this project 👍
